commands/command.go

```diff
@@ -50,6 +53,7 @@ type BaseCommand struct {
 	flagBaseURL       string
 	flagAtlasBaseURL  string
 	flagYes           bool
+	flagTelemetryOn   bool
```

Comment:
I prefer to avoid "default true" boolean flags, perhaps we switch this to `flagNoTelemetry` or `flagTelemetryOff` or something along those lines? `--no-telemetry` feels easiest to write out, but your call

---

also, let's move this field to live on the `telemetry.Service` struct definition.  you can still `set.BoolVar` the same with just with `&c.TelemetryService.NoTelemetry`, and avoid including it in the `SetFields` function signature